packages/components/src/components/assay/AssayResultDeleteModal.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import { useState } from 'react';
+
+import {
+    ConfirmModal,
+    createDeleteErrorNotification,
+    createDeleteSuccessNotification,
+    deleteRows,
+    Progress,
+    SchemaQuery,
+} from '../..';
+
+import { AssayResultDeleteConfirmModal } from './AssayResultDeleteConfirmModal';
```

Comment:
Both the `AssayResultDeleteConfirmModal` and the `AssayRunDeleteConfirmModal` don't warrant having their own class with separate stories as they are only intended to be used via their corresponding, non-confirming, modal. I'd recommend we consolidate those into their corresponding usage and remove them.

Reply:
Your refactor looks good and I agree they don't warrant their own components